packages/clay-form/stories/index.tsx

```diff
@@ -23,17 +26,15 @@ const ClayMultiSelectWithState = (props: any) => {
 	const [items, setItems] = React.useState(['one']);
 
 	return (
-		<div className="sheet">
-			<ClayInputWithMultiSelect
-				{...props}
-				inputName="myInput"
-				inputValue={value}
-				items={items}
-				onInputChange={setValue}
-				onItemsChange={setItems}
-				spritemap={spritemap}
-			/>
-		</div>
+		<ClayMultiSelectInput
```

Comment:
I intentionally wanted to leave this out so that we don't accidentally communicate anything in the storybook that might make devs think it looks like the out of the box. This is why I added the other story, the one that includes its composition with labels and buttons.

Reply:
Hmm yeah, that makes sense. Feel free to merge if you don't have more work on it.